app/javascript/components/carbon-charts/stackBarChart.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StackedBarChart } from '@carbon/charts-react';
+
+const StackBarChartGraph = ({ data }) => {
+  const StackBaroptions = {
```

Comment:
The other charts started with a lower case letter...should this be `stackBarOptions` ?

Reply:
Even simpler, can it just be `options`, and then make that consistent across all of the other charts as well?